shuup/core/pricing/_priceful.py

```diff
@@ -158,6 +159,10 @@ def unit_discount_amount(self):
 
     @property
     def tax_rate(self):
+        return ensure_decimal_places(self.raw_tax_rate)
```

Comment:
Shouldn't 0.1976500000123497 be formatted as "19.765%"? If it isn't then format_percent doesn't work correctly. (It should round the value with ROUND_HALF_UP or ROUND_HALF_EVEN, certainly not ROUND_UP which would cause the incorrect 19.766%)